VimeoNetworking/Sources/VimeoRequestSerializer.swift

```diff
@@ -96,47 +96,56 @@ final public class VimeoRequestSerializer: AFJSONRequestSerializer
     
     // MARK: Overrides
     
-    override public func request(withMethod method: String, urlString URLString: String, parameters: Any?, error: NSErrorPointer) -> NSMutableURLRequest
+    public override func request(withMethod method: String, urlString URLString: String, parameters: Any?, error: NSErrorPointer) -> NSMutableURLRequest
     {
-        var request = super.request(withMethod: method, urlString: URLString, parameters: parameters, error: error)
-       
-        request = self.setAuthorizationHeader(request: request)
+        var request = super.request(withMethod: method, urlString: URLString, parameters: parameters, error: error) as URLRequest
         
-        return request
+        request = self.requestWithAuthorizationHeader(fromRequest: request)
+        
+        return (request as NSURLRequest).mutableCopy() as! NSMutableURLRequest
```

Comment:
Are you sure that force casting in this case always succeeds? If not, let's do a proper guard/if let.